src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -20,4 +23,17 @@ public SnapshotBuilderSettings updateSnapshotBuilderSettings(
       UUID id, SnapshotBuilderSettings settings) {
     return snapshotBuilderSettingsDao.upsertSnapshotBuilderSettingsByDataset(id, settings);
   }
+
+  public SnapshotBuilderGetConceptsResponse getConceptsFromParent(Integer parentId) {
+    // TODO: Build real query - this should get the name and ID from the concept table, the count
+    // from the occurrence table, and the existence of children from the concept_ancestor table.
+    return new SnapshotBuilderGetConceptsResponse()
+        .result(
+            List.of(
+                new SnapshotBuilderConcept()
+                    .count(100)
+                    .name("Stub concept")
```

Comment:
I was trying to avoid putting in too much temporary code/structure here. I think this will enable us to test it without putting in too many fake concepts.